scripts/rageshake.go

```diff
@@ -37,8 +37,9 @@ func respond(code int, w http.ResponseWriter) {
 	w.Write([]byte("{}"))
 }
 
-func gzipAndSave(data []byte, fpath string) error {
-	fpath = filepath.Join("bugs", fpath)
+func gzipAndSave(data []byte, dirname, fpath string) error {
```

Comment:
It does. Go allows you to omit types on function declarations if subsequent ones are the same type. E.g:
```go
func foo(a int, b, c string) {
  // ...
}
```
Is the same as:
```go
func foo(a int, b string, c string) {
  // ...
}
```